test/.eslintrc.json

```diff
@@ -1,7 +1,8 @@
 {
   "rules": {
     "max-len": "off",
-    "object-curly-newline": "off"
+    "object-curly-newline": "off",
+    "func-names": "off"
```

Comment:
I modified a few tests to write shared values to `this` and removed arrow function literals to do this.

The use of arrow function literals is [explicitly discouraged][1] by mocha, but I'm happy to switch the tests back to using block-scoped variables and arrow literals if preferred. 

[1]: https://mochajs.org/#arrow-functions

Reply:
Hi @cjlarose Sorry for the delay, please revert the mocha test changes related to the arrow functions usage. I am aware of what mocha suggests and open to changes, however in order to make your change easier to merge right now, I need you to restore those changes.

Reply:
Will do! 👍